lib/economic/models/pdf.rb

```diff
@@ -3,12 +3,6 @@ module Models
     class Pdf < Economic::Model
       field :self
       field :download
-
-      def download_pdf
-        raise StandardError, "There is no download url set" if download.nil?
-
-        Net::HTTP.get(URI(download))
-      end
```

Comment:
Er det ikke en lidt breaking change?
Skulle den måske deprecates og bare kalde `Economic::Repos::Pdf` her?

Reply:
Det andet har aldrig virket, så jeg synes, det tæller som et fix.

Reply:
Er nu da ret sikker på at jeg bruger det

Reply:
<img width="783" alt="image" src="https://github.com/user-attachments/assets/8ef37611-1799-4387-ae42-f255228fa388" />

Reply:
Men det er det gamle API. Jeg tænkte mig ikke om ifht. testen, da jeg lavede det nye, så requestet blev aldrig authenticated.

Reply:
Ahhh! Got it, nu er jeg med 👍